source/documentation/gluster-hyperconverged/chap-Deploying_Hyperconverged.html.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Deploying oVirt and Gluster Hyperconverged
+---
+
+# Chapter 2: Deploying oVirt and Gluster Hyperconverged
+
+## Pre-requisites
+
+* You must have 3 freshly installed Enterprise Linux 7 hosts.
+
+* You must have atleast 2 interfaces on each of the hosts, so that the frontend and backend traffic can be separated out. Having only one network will cause the engine monitoring, client traffic, gluster I/O traffic to all run together and interfere each other. To segregate the backend network, the gluster cluster is formed using the backend network addresses, and the nodes are added to the engine using the frontend network address.
+
+* You must have a fully qualified domain name prepared for your Engine and the host. Forward and reverse lookup records must both be set in the DNS. The Engine should use the same subnet as the management network.
+
+* You must have configured passwordless ssh between the first host to itself and the other 2 hosts. This is needed for gdeploy to configure the 3 nodes. gdeploy uses Ansible playbooks and the ability to remotely execute commands is a pre-requisite.
+
+## Deploying on Enterprise Linux Hosts
```

Comment:
It might be worth adding a reference to installing on oVirt Node-based hosts as an option